README.md

```diff
@@ -59,6 +59,7 @@ Workstream Participant Agreement which is maintained as
 * Chris Wilson (Google) [@cwilso](https://github.com/cwilso)
 * Diego González (Microsoft) [@diekus](https://github.com/diekus)
 * Tantek Çelik (Mozilla) [@tantek](https://github.com/tantek)
+    * Alternate: Simon Pieters (Mozilla) [@zcorpan](https://github.com/zcorpan)
```

Comment:
I don't think we should list affiliation twice. I'm also not sure a separate bullet point is in order as we expect at most one Alternate. Perhaps after the Representative in parentheses?